hedyorg/hedy
# 🖊️ Add new indent error messages for level 8

Fixes #3270
Level 8 introduces indentation and allows single blocks while later levels allow nested blocks. Therefore, level 8 requires slightly different error messages.

Please consider whether the error messages could have better text.

**How to test**
Run Hedy locally and run the following snippets in level 8. The error messages should be meaningful and should not mention 'block in a block':
```
print 'test'
    print 'Hello folks'
```
```
repeat 5 times
print 'Hello folks'
```
```
repeat 5 times
    print 'test'
        print 'Hello folks'
```
```
repeat 5 times
    print 'test'
      print 'Hello folks'
```
```
repeat 5 times
    print 'test'
  print 'Hello folks'
```
